<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectDialog</class>
 <widget class="QDialog" name="CreateProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="projectDirectory">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="projectName">
       <property name="text">
        <string>Subdomain Project 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Project Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Create In:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
     background-color:  #E0E0E0;
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
	  
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }
</string>
     </property>
     <property name="title">
      <string>Full Domain Files</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="12" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link24">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Use Symbolic Link</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link14">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link15">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Required Files</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="chooseFort22">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>fort.23</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link13">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link20">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="fort13Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="fort19Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="fort23Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="fort10Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link11">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="chooseFort13">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="fort11Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QPushButton" name="chooseFort19">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QPushButton" name="chooseFort24">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QPushButton" name="chooseFort23">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="11" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link23">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link19">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>fort.24</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="fort20Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link22">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLineEdit" name="fort24Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Conditional Files</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="chooseFort10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="chooseFort15">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>fort.10</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>fort.20</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="chooseFort20">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>fort.22</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="fort15Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>fort.11</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>fort.19</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>fort.14</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="fort14Loc"/>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="fort22Loc">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QCheckBox" name="useSameDir">
          <property name="text">
           <string>Fill fields using files from fort.14 directory</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="3" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="link10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>fort.15</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>fort.13</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="chooseFort11">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="chooseFort14">
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="warnLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectName</tabstop>
  <tabstop>projectDirectory</tabstop>
  <tabstop>browseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
